### PR TITLE
SMS-6236: Verify Service SDK

### PR DESCRIPTION
**Pull request by ashutoshkumar-plivo**

### CODE REVIEW

File: spec/mocks/sessionGetResponse.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "api_id": "affc3af1-3ee3-4d8e-a577-317fabd4944e",
+    "session_uuid": "118a0352-6818-4cc4-99b4-635647730091",
+    "app_uuid": "bd772d6d-7882-4e77-b2b1-d84c7f73713a",
+    "alias": "verify_app_210",
+    "recipient": "918707046409",
```
Comment (narayana-plivo) [inline spec/mocks/sessionGetResponse.json:6]: you can hide this

File: lib/plivo/resources/verify_session.rb [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            def get(session_uuid)
+                perform_get(session_uuid)
+            end
+
+            def create(app_uuid = nil, recipient = nil,channel = nil,url = nil, method = nil)
+                value = app_uuid
+                if(value.is_a?(Hash))
+                    valid_param?(:app_uuid, value[:app_uuid], [String, Symbol], false)
+                    valid_param?(:recipient, value[:recipient], [Integer, String, Symbol], true)
+                    valid_param?(:channel, value[:channel], [String, Symbol], false)
```
Comment (narayana-plivo) [inline lib/plivo/resources/verify_session.rb:49]: value[:channel] ?

File: lib/plivo/resources/verify_session.rb [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                super
+            end
+
+            # @param [String] session_uuid
+            def get(session_uuid)
+                perform_get(session_uuid)
+            end
+
+            def create(app_uuid = nil, recipient = nil,channel = nil,url = nil, method = nil)
+                value = app_uuid
```
Comment (narayana-plivo) [inline lib/plivo/resources/verify_session.rb:45]: if your using hash, you can rename the app_uuid to params?

File: lib/plivo/resources/verify_session.rb [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                    }
+                else
+                    valid_param?(:session_uuid, session_uuid, [String, Symbol], true)
+                    valid_param?(:otp, otp, [String], true)  
+                    id = session_uuid
+                    params = {
+                        otp: otp
+                    }
+                end  
+                perform_create_with_id(id,params)
```
Comment (narayana-plivo) [inline lib/plivo/resources/verify_session.rb:132]: can we use perform_action? rather than adding new function

**Review by narayana-plivo — CHANGES_REQUESTED** [review]
please check review comment

File: lib/plivo/resources/verify_session.rb (hunk 2 above)
Comment (ashutoshkumar-plivo) [reply lib/plivo/resources/verify_session.rb]: removed hash

File: lib/plivo/resources/verify_session.rb (hunk 3 above)
Comment (ashutoshkumar-plivo) [reply lib/plivo/resources/verify_session.rb]: yes, removed hash params

File: lib/plivo/resources/verify_session.rb (hunk 4 above)
Comment (ashutoshkumar-plivo) [reply lib/plivo/resources/verify_session.rb]: yes

**Review by narayana-plivo — APPROVED** [review]
LGTM. @ashutoshkumar-plivo please resolve the merge conflict later

**Pull request merged**